Sources/_InternalTestSupport/GitRepositoryExtensions.swift

```diff
@@ -13,6 +13,7 @@
 import SourceControl
 
 import class Basics.AsyncProcess
+import class TSCBasic.Process
```

Comment:
suggestion: can we use `AsyncProcess` that is located in SwiftPM instead of having to rely on a TSC API?